<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cupat</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../static/icons/index.ico</normaloff>../static/icons/index.ico</iconset>
  </property>
  <widget class="QWidget" name="rootWidget">
   <layout class="QGridLayout" name="mainWindowWidgetGridLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
